examples/configs/classification_cifar10.yaml

```diff
@@ -89,8 +89,8 @@ seed_params:
   workers: true
 
 logger:
-  log_dir: '${oc.env:HOME}/.cache/torchok/logs/'
-  experiment_name: cifar10
+  log_dir: '${oc.env:HOME}/cifar10/logs'
+  experiment_name: resnet18
```

Comment:
It's not just `resnet18`, it's `cifar10_resnet18`

Reply:
No, cifar10 is located at higher hierarchy, therefore there is no need to duplicate information about cifar10

Reply:
I don't see an experiment `cifar10`, what do you mean?